sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_event_v1/query.sql

```diff
@@ -11,6 +11,7 @@ SELECT
     event_category = 'security.ui.protections'
     AND event_object = 'protection_report'
   ) AS n_viewed_protection_report,
+  mozfun.stats.mode_last(ARRAY_AGG(profile_group_id ORDER BY `timestamp`)) AS profile_group_id,
```

Comment:
Would this also affect the schema of `telemetry_derived.clients_last_seen_event_v1`?  That schema may need to be updated as well because it references itself

Reply:
Good catch, will update that also - thanks @BenWu